helix-term/src/commands.rs

```diff
@@ -249,9 +228,9 @@ pub fn move_next_word_start(cx: &mut Context) {
     let (view, doc) = cx.current();
     let text = doc.text().slice(..);
 
-    let selection = doc.selection(view.id).transform(|range| {
-        movement::move_next_word_start(text, range.head, count).unwrap_or(range)
```

Comment:
Why can we remove the `unwrap_or(range)`?

Reply:
I had to refactor the API to ensure the `count` worked. Before, passing a high count so that one of the moves failed returned a `None`. Now, the `move` functions will return the latest achievable range after iterating until it's not possible to move anymore.